api/migrations/0043_order_latitude_order_longitude_alter_order_bond_size.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 4.2.5 on 2023-10-03 20:12
+
```

Comment:
Sorry to bother again :D It would be best, in order to keep migration history clean, to rename this migration from `0043_order_latitude_order_longitude_alter_order_bond_size.py` to `0043_order_latitude_order_longitude.py`